spacy/pipeline/tagger.pyx

```diff
@@ -256,7 +258,8 @@ class Tagger(TrainablePipe):
         DOCS: https://spacy.io/api/tagger#get_loss
         """
         validate_examples(examples, "Tagger.get_loss")
-        loss_func = SequenceCategoricalCrossentropy(names=self.labels, normalize=False, neg_prefix=self.cfg["neg_prefix"])
+        self.cfg["label_smoothing"] = 0.05 if self.cfg["label_smoothing"] else 0.0
+        loss_func = SequenceCategoricalCrossentropy(names=self.labels, normalize=False, neg_prefix=self.cfg["neg_prefix"], label_smoothing=self.cfg["label_smoothing"])
```

Comment:
```suggestion
        label_smoothing = 0.05 if self.cfg["label_smoothing"] else 0.0
        loss_func = SequenceCategoricalCrossentropy(names=self.labels, normalize=False, neg_prefix=self.cfg["neg_prefix"], label_smoothing=label_smoothing)
```

Reply:
I realized that this might not be the best suggestion looking at the larger picture. If this is what you want to do every time, it would probably make sense to do this in `__init__` instead. (I'm not thrilled at having the same internal setting name with a different type in `cfg`, though.)

Reply:
> (I'm not thrilled at having the same internal setting name with a different type in cfg, though.)

Yeah this is why I don't set it in `__init__`. I don't think there's any actual problem having it here, at least not yet.

Reply:
If it's something that only needs to be done once, it's clearer in the future if it's in `__init__` and not buried in another method. Then maybe call the internal value something else like `label_smoothing_value`. (This is still kind of dissatisfying given the `thinc` API, though.)

Reply:
I agree it's nicer to set this in the constructor. If we want to expose this differently than a bool in the future, it's nice if the stored label smoothing is just a `float` and not some union type.

Reply:
Okay, so I think the consensus is to just store this as a `float` instead of a `bool` or a union - now that we've decided to have it enabled by default, it makes more sense to use `0.05` as a default value, and allow people to set it to `0.0`/whatever other value they had in mind.